plugins/adr/src/components/AdrReader/AdrReader.tsx

```diff
@@ -45,8 +45,18 @@ export const AdrReader = (props: {
   const scmIntegrations = useApi(scmIntegrationsApiRef);
   const adrApi = useApi(adrApiRef);
   const adrLocationUrl = getAdrLocationUrl(entity, scmIntegrations);
+  let url = `${adrLocationUrl.replace(/\/$/, '')}`;
```

Comment:
I wonder if this fix is in the wrong place.

Should this fix be in the `resolveUrl` for the `Azure` integration instead?\

https://github.com/backstage/backstage/blob/5f1a92b9f19fd98e218637a4bf40ddc3e3fe7a6b/packages/integration/src/azure/AzureIntegration.ts#L58-L91

Reply:
@benjdlambert Since I don't see any other use-case for this path append in Azure DevOps Source URL expect the `ADR plugin`. I added this in ADR plugin itself. I will check it in the AzureIntegration too.

Reply:
Ah right, now I understand.

I think then it should be passed through to here then right? https://github.com/backstage/backstage/blob/2b4513abb7842a75d693ce4c46b9da98ecf75499/plugins/adr-common/src/index.ts#L66

Reply:
@kuangp Yes you are right!, your solution is actually working 🚀.  Apologies It my bad typo will leads the failure while testing so I decided to go with different approach. I should have verified once more during the test.

I will do the refactor and make is in the appropriate location `getAdrLocationUrl()` and update here.

Thanks again @kuangp to point it out 👍

Reply:
@benjdlambert @kuangp 
I updated the logic to `getAdrLocationUrl()` and Just to make sure I test with `Azure DevOps`, `GitHub` & `BitBucket Cloud` all good with the results.